README.md

```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
 print(json.dumps(report["data stats"][0], indent=4))
 ```
 
+### Unstructured profiler
+In addition to structure profiler, DataProfiler provides unstructured profiling for the TextData object or string. Unstructured profile also works with list(string), pd.Series(string) or pd.DataFrame(string) given profiler_type option specified as `unstructured`. Below is an example of unstructured profile with a text file. 
```

Comment:
`In addition to structure profiler` -> `In addition to the structured profiler`

Reply:
`Below is an example of unstructured profile` -> `Below is an example of the unstructured profiler`

Reply:
fixed